remote-query/remote-query-client/src/test/java/org/infinispan/query/remote/client/BaseProtoStreamMarshallerTest.java

```diff
@@ -5,6 +5,7 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
+import org.infinispan.commons.marshall.protostream.BaseProtoStreamMarshaller;
```

Comment:
the test is in hr client and the implementation in commons :)

Reply:
Good spot!